make_UPD_igv.py

```diff
@@ -0,0 +1,176 @@
+            vcfs[sampleid] = vcf  
+    for sample in families:
+        family = samples[sample]['family']
+
+        if sample not in vcfs:  # PED file includes all samples of entire run, not only project. Skip if child VCF is not present in current analysis.
+            continue
+    
+        if args.sample_id not in sample:
+            continue
+
+        child = parse_vcf(vcfs[sample])
+        father = dict(parse_vcf(vcfs[families[sample][0]]))  ## father always first item
+        mother = dict(parse_vcf(vcfs[families[sample][1]]))  ## mother always second item
+
+        output_file = open("{}_{}.igv".format(args.run_id, family), 'w')
+        output_file.write("#track type=igv name=Pat/Mat_segments color=204,204,0 altColor=0,100,224 graphType=bar windowingFunction=none maxHeightPixels=50 viewLimits=-1,1\n")
+        chromosome = "1" 
+        start = 0
+        for variant in child:
+            if variant[0] in father and variant[0] in mother:  # position is called in both father and mother
+                position, child_geno, father_geno, mother_geno = variant[0], variant[1], father[variant[0]], mother[variant[0]]
+                label = '' 
+                score = ''
+                chrom = str(position.split("_")[0])
+                if chrom != chromosome:
+                    start = 0  
+                    chromosome = chrom
+
+                snv_pos = int(position.split("_")[1])
+                start = start
+                end = snv_pos
+
+                genotype_conversion = {"0/0":"homref","0/1":"het", "1/1":"homvar"}
+
+                genotype_score = {
+                    "homref_homvar_homref": ["patIso", 1],
+                    "homref_homvar_homvar": ["matIso", -1],
+                    "homref_homvar_het": ["normal", -0.2],
+                    "homvar_homref_homvar": ["patIso", 1],
+                    "homvar_homref_homref": ["matIso", -1],
+                    "homvar_homref_het": ["normal", 0.2],
+                    "homvar_het_homref": ["matIso", -1],
+                    "het_homvar_homref": ["patIso", 1],
+                    "het_homref_het": ["patHet", 1],
+                    "homref_het_het": ["matHet", -1],
+                    "het_homref_homvar": ["patIso", 1], 
+                    "homref_het_homvat": ["matIso", -1]
+                    }
+
+                if father_geno in genotype_conversion and mother_geno in genotype_conversion and child_geno in genotype_conversion:
+                    genotype = "{}_{}_{}".format(genotype_conversion[father_geno], genotype_conversion[mother_geno],genotype_conversion[child_geno])
+                    if genotype in genotype_score:
+                        label = genotype_score[genotype][0]
+                        score = genotype_score[genotype][1]
+
+                binsize = 0
+            
+                if int(end - start) > args.maxlocus:
+                    start = end - args.maxlocus
+
+                if label: 
```

Comment:
if no label, there is no outputfile?

Reply:
If no label, the variant position is not **printed** in output. 
Depending on the inheritence patern, the variant position should or should not be printed in output.

i.e. when child is 0/1, father is 0/1, and mother is 0/1, there position is not informative for the UPD track